css/src/mixins/gradient.scss

```diff
@@ -1,10 +1,10 @@
-@mixin gradient-text($startColor, $endColor) {
+@mixin gradient-text($startColor, $endColor, $direction: 90deg) {
```

Comment:
Looking at line 90 in gradient.scss, should `$direction` here be `$angle` instead?

Reply:
updated!